README.md

```diff
@@ -1,3 +1,110 @@
 # `winston-seq`
 
-A Winston v3 transport for Seq; currently work-in-progress.
+A [Winston](https://github.com/winstonjs/winston) v3 transport that sends structured logs to the [Seq Log Server](https://datalust.co/seq).
+
+![Structured logging with Seq](assets/seq-log-search-feature-2220w.gif)
+
+## Install @datalust/winston-seq
+
+```sh
+$ npm install @datalust/winston-seq winston
+# Or with yarn
+$ yarn add @datalust/winston-seq winston
+```
+
+## Configure Logging
+
+```ts
+const winston = require('winston');
+const seq = require('@datalust/winston-seq');
```

Comment:
Since the convention seems to be for Winston transports to be classes, maybe we could consider exporting `Seq` or `SeqTransport` from the module? E.g.:

```ts
const { SeqTransport } = require('@datalust/winston-seq');
// new SeqTransport({...})
```

I've dug around some other transports, it seems like `new seq.SeqTransport()` or `new Seq(...)` would both be reasonably in sync with the others.

Reply:
The 'Maintained by winston contributors' ones monkey patch winston.transports which I'd prefer not to do, and does not seem common among the community transports. They seem evenly split between exporting the transport constructor and exporting an object with a property that is the transport constructor. The idiomatic node way is to export just a function if that is all your module is, so I suggest we go that way.

Reply:
`const SeqTransport = require('@datalust/winston-seq');` in the examples?

Uppercase is better for a class name.